src/Uno.UI/UI/Xaml/FrameworkElement.cs

```diff
@@ -740,10 +735,7 @@ private void OnIsEnabledChanged(DependencyPropertyChangedEventArgs args)
 		{
 			UpdateHitTest();
 
-			_isEnabledChangedEventArgs ??= new IsEnabledChangedEventArgs();
-			_isEnabledChangedEventArgs.SourceEvent = args;
-
-			OnIsEnabledChanged(_isEnabledChangedEventArgs);
```

Comment:
This part of the code is what led me to this refactoring. It was confusing why we set the needed info to a static `_isEnabledChangedEventArgs` to pass it when we can directly pass it and make the code more readable.

Reply:
The field is not useful. I would suggest changing the `IsEnabledChangedEventArgs` to be a read-only `struct` and just create it and pass it forward. This way we will keep the type for porting parity and avoid potential weird issues coming from it being a static field

Reply:
Agreed. Let's keep the code unmodified for source parity, and change the type to a struct.

Reply:
@Youssef1313 if changing to a struct does not give much benefit, let's close this PR.